public/css/main.less

```diff
@@ -526,7 +526,8 @@ thead {
 }
 
 form.code {
-  font-size: 22px;
+  font-size: 16px;
```

Comment:
I picked 16px because it's equivalent to 12pt on most screens, and it's what I expect to see typically in a code editor. At 22px it looks HUGE in my opinion.